client/packages/react-ui/src/components/InscriptionEditor.tsx

```diff
@@ -1,15 +1,15 @@
 import { Separator } from '@radix-ui/react-separator';
 import { Tabs, TabsContent, TabsList, TabsTrigger } from '@radix-ui/react-tabs';
 import { memo } from 'react';
-import editorIcon from '../icons/user-dialog.svg';
-import './InscriptionEditor.css';
+import editorIcon from '../../style/icons/user-dialog.svg';
+import '../../style/InscriptionEditor.css';
 import { EditorProps } from './props/editor';
 
 const Header = (props: EditorProps) => (
   <>
     <div className='header'>
       <div className='header-title'>
-        <div className='header-editor'>{props.title}</div>
+        <div className='header-editor'>rf{props.title}</div>
```

Comment:
?

Reply:
Sorry, this was committed by mistake. I did a 'watch and update' test and seems like this sneaked into the commit. Sorry for that!

Reply:
No problem 😀